### PR TITLE
Replace all occurrences of Vector by AbstractVector in plot commands

### LINKED ISSUE #73: Support for other AbstractArray
Thanks for the package, it is awesome. I tried to create a barplot today with a `MVector` containing the values and realized that the function are only defined in terms of `Vector`. Would it be a good idea to generalize the signature of the plot commands to work with any `AbstractArray` or `AbstractVector`?

### PR DESCRIPTION
**Pull request by juliohm**
Fix #73 

The outputs in the test suite looked correct to me, but I appreciate if you can double check.

### CODE REVIEW

File: src/interface/densityplot.jl [hunk 1]
```diff
@@ -88,23 +88,23 @@ See also
...
-        x::Vector{F},
-        y::Vector{R};
+        x::AbstractVector{F},
+        y::AbstractVector{R};
         color::Symbol = :white,
         args...)
-    X = convert(Vector{Float64}, x)
-    Y = convert(Vector{Float64}, y)
+    X = convert(AbstractVector{Float64}, x)
+    Y = convert(AbstractVector{Float64}, y)
```
Comment (Evizero) [inline src/interface/densityplot.jl]: this doesn't look like a good idea. let's leave this as `Vector` for `convert`

File: src/interface/densityplot.jl [hunk 2]
```diff
@@ -88,23 +88,23 @@ See also
...
```
Comment (Evizero) [inline src/interface/densityplot.jl]: same here (leave `Vector`)

**Comment by juliohm** [comment]
Oh yeah, that wasn't intended, will fix it tomorrow morning. Calling the
day here...

**Comment by juliohm** [comment]
Done :+1:

**Comment by Evizero** [comment]
Thanks!

**Pull request merged**